ios/RNSentry.mm

```diff
@@ -633,10 +663,23 @@ - (NSDictionary*) fetchNativeStackFramesBy: (NSArray<NSNumber*>*)instructionsAdd
         }
 #endif
 
-        return @{ @"profile": data };
+        return @{
+          @"profile": data,
+          @"nativeProfile": nativeProfile,
```

Comment:
`data` and `nativeProfile` can both be `nil`, so we should check them before inserting into this dict literal, or it could crash.

Reply:
Thanks.